ppr-ui/src/composables/useInputRules.ts

```diff
@@ -6,6 +6,12 @@ export const useInputRules = () => {
     ]
   }
 
+  const isStringOrNumber = (): Array<Function> => {
+    return [
+      v => (v ? /^[a-zA-Z0-9_ ]*$/g.test(v) : true) || 'Invalid characters'
```

Comment:
If the intent is to have alphanumeric values, then we should probably exclude underscore and space.

Reply:
UX request to allow white space but to 'handle' it. 
Ie why we trim and replace all white space when setting the serial number.